integreat_cms/cms/migrations/0089_add_language_color.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 4.2.10 on 2024-04-03 08:12
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Migration is needed when a change is made in a model. Use `./tools/run.sh` and a migration file will be generated 😺  (or at least I do it in this way)
If multiple of migration files have been generated during implementation, delete them all beforehand 👍

Reply:
@MizukiTemma I also wondered whether I need to delete the old ones or not. Thank you for your commit and info! :)